llama_stack/apis/memory/client.py

```diff
@@ -75,14 +75,22 @@ async def query_documents(
 async def run_main(host: str, port: int, stream: bool):
     banks_client = MemoryBanksClient(f"http://{host}:{port}")
 
-    bank = VectorMemoryBankDef(
+    bank = VectorMemoryBank(
```

Comment:
this place now has both `VectorMemoryBank` (that object actually is not used) and has `VectorMemoryBankParams` -- only the latter should be there?